pep-0514.txt

```diff
@@ -200,12 +240,16 @@ always named ``InstallPath``, and the default value must match ``sys.prefix``::
 If a string value named ``ExecutablePath`` exists, it must be a path to the
 ``python.exe`` (or equivalent) executable. Otherwise, the interpreter executable
 is assumed to be called ``python.exe`` and exist in the directory referenced by
-the default value.
+the ``(Default)`` value.
```

Comment:
To confirm - the absolute minimum a correctly registered installation must provide is:

```
    HKEY_XXX\Software\Python\<Company>\<Tag>\InstallPath
        (Default) = "<sys.prefix>"
```

Everything else is optional?